arb_os/evmCallStack.mini

```diff
@@ -783,6 +785,8 @@ public impure func evmCallStack_returnFromCall(
                             Some(gasUsage),
                             feeStats,
                         );
+
+                        arbStatistics_addContract();
```

Comment:
This covers externally created contracts, but does it cover `CREATE` and `CREATE2`?

Reply:
Added coverage for CREATE and CREATE2